src/meta-srv/src/metasrv.rs

```diff
@@ -246,6 +248,13 @@ impl MetaSrv {
                 .context(RecoverProcedureSnafu)?;
         }
 
+        // FIXME (paomian)
```

Comment:
It should be resolved in this pr.
I'll revise here after I get the specifics with the appropriate people.